Source/Particles/MultiParticleContainer.cpp

```diff
@@ -397,21 +397,42 @@ MultiParticleContainer::AllocData ()
 void
 MultiParticleContainer::InitData ()
 {
+    InitMultiPhysicsModules();
+
     for (auto& pc : allcontainers) {
         pc->InitData();
     }
     pc_tmp->InitData();
+
+}
+
+void
+MultiParticleContainer::PostRestart ()
```

Comment:
I moved the function `PostRestart` closer to the function `InitData` in this file, to make it easier to see the similarities between these functions.